### ISSUE TITLE
MacOSX cannot delete characters in "Entry name" field.

### ISSUE THREAD

**Issue by Gnappuraz** [Ruteri/hdpwm#2]
Trying to delete character while filling the "Entry name" field causes `^?` to be printed and later saved instead.

I could only verify this on MacOSX 10.15 Catalina.

<img width="170" alt="Screenshot 2019-11-06 at 16 23 35" src="https://user-images.githubusercontent.com/756391/68311409-cc98d180-00b1-11ea-9445-7e65456d2fe6.png">

**Comment by Ruteri** [comment]
Since I cannot reproduce it on Ubuntu 18, it's most likely that the key passed into `InputHandler::process_key` is not `KEY_BACKSPACE` when you press backspace (otherwise it would do _something_ but printing the `^?` key).

**Comment by Gnappuraz** [comment]
I agree, I was trying to debug what is happening there but ncurses don't make it easy.
